e2e/src/app.e2e-spec.ts

```diff
@@ -34,24 +39,53 @@ describe('test chatbot scenarion 1 - new user', () => {
     expect(lastMessage).toContain(STR.ben);
     expect(lastMessage).not.toContain(STR.bat);
   });
+});
 
-  fit('[Scenario] should fill report for a new user ', async () => {
-    const asnwers: Array<AnswerTestDataType> = [
-      0, // ben
-      24, // 24 yrs
-      STR.telAvivYaffo,
-      STR.herzl,
-    ];
-    const alias = `${STR.ben} 24 ${STR.from}${STR.herzl} ${STR.telAvivYaffo}`;
+describe('[Flow] Reporting For a first address on a device', () => {
+  let page: AppPage;
+  // ben 24 from herzl tel aviv yaffo
+  const asnwers: Array<AnswerTestDataType> = [0, 24, STR.telAvivYaffo, STR.herzl];
+  const alias = `${STR.ben} 24 ${STR.from}${STR.herzl} ${STR.telAvivYaffo}`;
 
-    await page.navigateTo();
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should fill report for a new user ', async () => {
+    page.navigateTo();
     await simulateChatFlow(page, STR.reportSent, asnwers);
     // report sent - get wouldSend object
     browser.executeScript('return window.wouldSend;').then(function (wouldSend) {
-      console.log('*** wouldSend *** ');
-      console.log(wouldSend);
+      log(`*** wouldSend[alias] : ${wouldSend['alias']} *** `);
       expect(wouldSend['alias']).toBeDefined();
       expect(wouldSend['alias']).toEqual(alias);
     });
   });
 });
+
+describe('[Flow] Returning user can not start a new report in the same day', () => {
+  let page: AppPage;
+  // bat 30 from balfour raanana
+  const asnwers: Array<AnswerTestDataType> = [1, 30, STR.raanana, STR.balfour];
+  const alias = `${STR.bat} 30 ${STR.from}${STR.balfour} ${STR.raanana}`;
+
+  beforeEach(async() => {
+    page = new AppPage();
+    // fill random user report for 'alias'
+    page.navigateTo();
+    await simulateChatFlow(page, STR.reportSent, asnwers);
+    page = new AppPage();
+  });
+
+  fit('should not allow another report on same day', async () => {
```

Comment:
it or fit?

Reply:
you are right, changing it in the next PR